deRSE23/README.md

```diff
@@ -0,0 +1,11 @@
+# deRSE23 Submission
+
+Abstract (and slides) for the [deRSE23](https://de-rse23.sciencesconf.org/) conference.
+
+* Title: Introducing the Fortran-lang community
+* Presenter: Ivan Pribec
+* Co-authors: Vincent Magnin, Asdrubal Lozada-Blanco, Ondrej Certik, Jeremie Vandenplas, Milan Curcic
+
+## Abstract
+
+A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering, and evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, an open-source community Fortran-lang has formed in 2019. Here, we present the efforts of Fortran-lang such as the Fortran package manager, standard library, website, LFortran compiler and other tools. Fortran-lang aims to provide a central place for Fortran users, complementing the language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```

Comment:
Just some punctuation and rewording suggestions.

```suggestion
A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering and the evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. An open-source community Fortran-lang formed in 2019 to address this challenge. Here, we present the efforts of Fortran-lang: such as the Fortran package manager, standard library, website, LFortran compiler, and other tools. Fortran-lang aims to provide a central place for Fortran users, complementing language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```

Reply:
Thanks Sebastian. I don't think a colon is necessary after "such as". With respect to the third sentence, I would prefer to keep the current version which is 5 characters shorter, and states we are providing a solution at the beginning of the sentence. It also doesn't repeat "challenge" twice. Let me know if the latest version works for you.

Reply:
'An open-source community Fortran-lang': The title of the Discourse `Fortran Discourse - Fortran open source community` and all initiated projects are open source. But the community also includes developers and users of proprietary code (e.g. the Intel compiler).